xml/ay_bigfile.xml

```diff
@@ -11223,20 +11236,20 @@ echo "Testing the pre script" &gt; /tmp/pre-script_out.txt
         <entry>
          <para>
           If set to <literal>false</literal>, the installed system remains
-          mounted at <filename>/mnt</filename> and no chroot happens. The
-          boot loader is not installed either at this stage. Setting it to
-          <literal>true</literal> means, a chroot into
-          <filename>/mnt</filename> is performed, where the installed system
-          is mounted. The boot loader is installed, and if you want to
-          change anything in the installed system, you do not need to use the
+          mounted at <filename>/mnt</filename> and no chroot happens. Also, the
+          boot loader is not installed at this stage. Setting it to
+          <literal>true</literal> means that a chroot into 
+          <filename>/mnt</filename>, where the installed system is mounted, is
```

Comment:
To avoid the insertion 'where the installed system is mounted', I would suggest to move this part to the end of the sentence: 'Setting it to [...] means that a chroot into ```<filename>/mnt</filename>``` is performed, where the installed system is mounted.'

Reply:
That is what I was trying to fix. :-)

Reply:
Reworded to avoid the parenthetical phrase.